inst/shinyApp/drawCellShiny/server.R

```diff
@@ -119,6 +130,9 @@ function(input, output) {
     {
       sc_id(NULL)
       subcellular_colours(list("SL0000" = "#56B4E9"))
+      # This step is necessary. Otherwise, clearing the colors will reset the cell to the
+      # default  animal cell
+      subcellular_colours(find_unique_subcellular_location(input$cell_type))
```

Comment:
It might need the `drawCell:::` since `find_unique_subcellular_locaiton` is not an exported function.